src/pastelid/common.h

```diff
@@ -7,62 +7,73 @@
 #include <openssl/pem.h>
 #include <openssl/err.h>
 #include <openssl/crypto.h>
+#include "vector_types.h"
 
 namespace ed_crypto {
 
     static constexpr int OK = 1;
-	
-	inline std::string Base64_Encode(const unsigned char *in, size_t len)
+    static constexpr auto BASE64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+	inline std::string Base64_Encode(const unsigned char *in, const size_t len) noexcept
     {
         std::string out;
+        out.reserve(static_cast<size_t>(ceil(len / 3) * 4));
         int val=0, valb=-6;
```

Comment:
missing spaces

Reply:
If you see one line change - it means that I "changed one line only" instead of rewriting the whole function.
In this case I just added string reservation to improve base64 encoding performance.

Reply:
ok. good then